diff --git a/node_modules/ytdl-core/lib/info-extras.js b/node_modules/ytdl-core/lib/info-extras.js
index e735a4a..e4811c6 100644
--- a/node_modules/ytdl-core/lib/info-extras.js
+++ b/node_modules/ytdl-core/lib/info-extras.js
@@ -236,8 +236,7 @@ exports.getLikes = info => {
     let contents = info.response.contents.twoColumnWatchNextResults.results.results.contents;
     let video = contents.find(r => r.videoPrimaryInfoRenderer);
     let buttons = video.videoPrimaryInfoRenderer.videoActions.menuRenderer.topLevelButtons;
-    let like = buttons.find(b => b.toggleButtonRenderer &&
-      b.toggleButtonRenderer.defaultIcon.iconType === 'LIKE');
+    let like = buttons.find(b => b.segmentedLikeDislikeButtonRenderer).segmentedLikeDislikeButtonRenderer.likeButton;
     return parseInt(like.toggleButtonRenderer.defaultText.accessibility.accessibilityData.label.replace(/\D+/g, ''));
   } catch (err) {
     return null;
diff --git a/node_modules/ytdl-core/lib/utils.js b/node_modules/ytdl-core/lib/utils.js
index 705c8dd..143ef1f 100644
--- a/node_modules/ytdl-core/lib/utils.js
+++ b/node_modules/ytdl-core/lib/utils.js
@@ -60,7 +60,7 @@ const ESCAPING_SEQUENZES = [
   { start: "'", end: "'" },
   { start: '`', end: '`' },
   // RegeEx
-  { start: '/', end: '/', startPrefix: /(^|[[{:;,])\s?$/ },
+  { start: '/', end: '/', startPrefix: /(^|[[{:;,/])\s?$/ },
 ];
 
 /**
